npm/webpack-dev-server/src/makeWebpackConfig.ts

```diff
@@ -183,12 +146,5 @@ export async function makeWebpackConfig (
 
   debug('Merged webpack config %o', mergedConfig)
 
-  if (process.env.WEBPACK_PERF_MEASURE) {
```

Comment:
We don't have this documented, and it doesn't work for many Webpack 5 apps so it's getting removed.